src/Lecture2.hs

```diff
@@ -178,7 +220,9 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing [_] = True
+isIncreasing (x:y:xs) = (if (x <= y) then True else False) && isIncreasing xs
```

Comment:
This can be simplified a bit 🧑‍🔬 
```suggestion
isIncreasing (x:y:xs) = x <= y && isIncreasing xs
```

Also, I see that this function is incorrect 🤔 
It will return `True` on the list `[1, 2, 1]` but it should return `False`. Probably, it's a good test to add to the test-suite